nucypher/keystore/keypairs.py

```diff
@@ -3,19 +3,20 @@
 import sha3
 from OpenSSL.SSL import TLSv1_2_METHOD
 from OpenSSL.crypto import X509
-from cryptography.hazmat.primitives.asymmetric import ec
 from constant_sorrow import constants
+from cryptography.hazmat.primitives.asymmetric import ec
 from hendrix.deploy.tls import HendrixDeployTLS
 from hendrix.facilities.services import ExistingKeyTLSContextFactory
-from nucypher.crypto import api as API
-from nucypher.crypto.api import generate_self_signed_certificate
-from nucypher.crypto.kits import MessageKit
-from nucypher.crypto.signing import SignatureStamp, StrangerStamp
 from umbral import pre
 from umbral.config import default_curve
 from umbral.keys import UmbralPrivateKey, UmbralPublicKey
 from umbral.signing import Signature, Signer
 
+from nucypher.crypto import api as API
+from nucypher.crypto.api import generate_self_signed_certificate
+from nucypher.crypto.kits import MessageKit
+from nucypher.crypto.signing import SignatureStamp, StrangerStamp
+
 
 class Keypair(object):
```

Comment:
Remove the `certificate` stuffs here.

Reply:
Yeah, definitely.  That's not supposed to be there.  But let's get this merged so we're flat again and then of course I'll take care of it.

Reply:
100% agree

Reply:
#395